### PR TITLE
Update quickstart-setup-auto-enrollment.md

### PR DESCRIPTION
**Pull request by msbemba**
Added the important note to the document stating that For Windows BYOD devices, the MAM user scope takes precedence if both the MAM user scope and the MDM user scope (automatic MDM enrollment) are enabled for all users

fixes#https://github.com/MicrosoftDocs/memdocs/issues/3342

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/enrollment/quickstart-setup-auto-enrollment.md [hunk 1]
```diff
@@ -66,12 +66,18 @@ For this example, you'll use MDM enrollment so that both corporate and bring-you
...
 
-6. For **MAM User scope**, select **Some** to manage data on your workforce's devices.  
-
+6. For **MAM User scope**, select **None**.  
 7. Choose **Select groups** > **Contoso Testers** > **Select** as the assigned group. 
 8. Use the default values for the remaining configuration values on the page.    
 9. Choose **Save**.
 
+> [!IMPORTANT]
+> For Windows BYOD devices, the MAM user scope takes precedence if both the MAM user scope and the MDM user scope (automatic MDM enrollment) are enabled for all users (or the same groups of users). The device will not be MDM enrolled, and Windows Information Protection (WIP) Policies will be applied if you have configured them.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/quickstart-setup-auto-enrollment.md:75]:
```suggestion
> For Windows BYOD devices, the MAM user scope takes precedence if both the MAM user scope and the MDM user scope (automatic MDM enrollment) are enabled for all users (or the same groups of users). The device will not be MDM enrolled, and Windows Information Protection (WIP) policies will be applied if you have configured them.
```

File: memdocs/intune/enrollment/quickstart-setup-auto-enrollment.md [hunk 2]
```diff
@@ -66,12 +66,18 @@ For this example, you'll use MDM enrollment so that both corporate and bring-you
...
+
+If your intent is to enable automatic enrollment for Windows BYOD devices to an MDM: configure the MDM user scope to All (or Some, and specify a group) and configure the MAM user scope to None (or Some, and specify a group – ensuring that users are not members of a group targeted by both MDM and MAM user scopes).
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/quickstart-setup-auto-enrollment.md:77]:
```suggestion
If your intent is to enable automatic enrollment for Windows BYOD devices to an MDM, configure the MDM user scope to All (or Some, and specify a group) and configure the MAM user scope to None (or Some, and specify a group–ensuring that users are not members of a group targeted by both MDM and MAM user scopes).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please signoff this PR. Thanks

**Comment by lenewsad** [comment]
#sign-off

**Pull request merged**